### ISSUE TITLE
Mechanism/Motor CheckSystems

### ISSUE THREAD

**Issue by allengregoryiv** [Spectrum3847/SpectrumTraining#16]
Mechanism abstract class gets a 

Boolean checkSystem() method that runs the motor and checks it's currents, and velocity/position measurements to make sure they are working correctly.

**Comment by Daniel-J101** [comment]
checking the current/velocity/position could be done inside the method, but shouldn't this be used inside of a command in order to wait for the motor to run before checking measurements?

**Comment by allengregoryiv** [comment]
Yes I think you are right, check system should be a command that is run on each mechanism.
